composer/workflows/dataflowtemplateoperator_create_dataset_and_table_helper.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is a helper function to the Dataflow Template Operator Tutorial.
+# It helps the user set up a BigQuery dataset and table that is needed
+# for the tutorial.
+
+import argparse
+
+# [START composer_dataflow_dataset_table_creation]
+
+# Make sure to follow the quickstart setup instructions beforehand.
+# See instructions here:
+# https://cloud.google.com/bigquery/docs/quickstarts/quickstart-client-libraries
+
+# Before running the sample, be sure to install the bigquery library
+# in your local environment by running pip install google.cloud.bigquery
+
+# If you choose to download the library, you can also run the sample with the
+#  following command:
+#    python dataflowtemplateoperator_create_dataset_and_table_helper.py \
+#    PROJECT_ID \
+#    LOCATION
+#   where `PROJECT_ID` is your Google Cloud Project ID and `LOCATION` is where you
+#   want the location of your BigQuery dataset.
+
+from google.cloud import bigquery
+
+
+def create_dataset_and_table(project, location, dataset_name):
```

Comment:
I noticed the location is not actually used in this function. Do we still want it here or is something missing?

Reply:
good catch! Sorry, took it out during testing.